### PR TITLE
Added some comments in readme.md file

### PR DESCRIPTION
**Pull request by Pooja-Gangavathi**
Added some comments in readme.md file

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Opencartdemo website login functionality Cypress automation
+# Opencartdemo website login functionality Cypress automation <!-- Title can be short and concise -->
+
```
Comment (Pooja-Gangavathi) [inline readme.md:2]: Title can be short and concise. You can remove the mentioned tags

File: readme.md [hunk 2]
```diff
@@ -23,7 +24,8 @@ Above setup must be completed before executing the testcases
    ```
 
    ```bash
-   cd CypressAutomatio_demo.opencart_Loginfunctionality
+   cd CypressAutomatio_demo.opencart_Loginfunctionality <!-- Command not working -->
+
```
Comment (Pooja-Gangavathi) [inline readme.md:28]: Please check the above command.

**Review by Pooja-Gangavathi — COMMENTED** [review]
Reviewed

File: readme.md (hunk 1 above)
Comment (nidhisingh87) [reply readme.md:2]: Noted and corrected.

File: readme.md (hunk 2 above)
Comment (nidhisingh87) [reply readme.md:28]: Rectified the command.

**Comment by nidhisingh87** [comment]
Gone through with the comments and made corrections

**Pull request closed without merging**